docs/README.md

```diff
@@ -55,6 +55,8 @@ With a clear project scope, lightweight governance model, and clean lines of
 separation between pluggable components, the Knative project establishes an
 efficient contributor workflow.
 
+While not part of a software foundation, Knative provides contributors with a clear project scope, open governance with multiple companies represented, and clean lines of separation between pluggable components. The Knative project strives for an efficient contributor workflow.
```

Comment:
There is some repetition in the paragraphs of this section. We should try to combine these statements? Also, should our formal "status" be explained in the "knative/community" repo? Maybe in this topic: https://github.com/knative/community/blob/master/CONTRIBUTING.md

Reply:
Good catch, the previous paragraph should be deleted. I'll do that now.